lib/Extension/PHPUnit/CodeTransform/Domain/Refactor/GenerateTestMethods.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Phpactor\Extension\PHPUnit\CodeTransform\Domain\Refactor;
+
+use Generator;
+use PHPUnit\Framework\TestCase;
+use Phpactor\CodeBuilder\Domain\Builder\SourceCodeBuilder;
+use Phpactor\CodeBuilder\Domain\Code;
+use Phpactor\CodeBuilder\Domain\Updater;
+use Phpactor\TextDocument\ByteOffset;
+use Phpactor\TextDocument\TextDocument;
+use Phpactor\TextDocument\TextDocumentEdits;
+use Phpactor\TextDocument\TextDocumentUri;
+use Phpactor\TextDocument\WorkspaceEdits;
+use Phpactor\WorseReflection\Core\ClassName;
+use Phpactor\WorseReflection\Core\Exception\NotFound;
+use Phpactor\WorseReflection\Core\Reflection\ReflectionClass;
+use Phpactor\CodeTransform\Domain\SourceCode;
+use Phpactor\WorseReflection\Reflector;
+
+
+class GenerateTestMethods /* implements GenerateMethod */
```

Comment:
this isn't "domain" a it knows about worse reflection. and this type of abstraction isn't really worth it for extensions. Putting itin `lib/Extension/PHPUnit/CodeTransform/GenerateMethods.php` would be fine.